### PR TITLE
Removing rabbit completely from collex

### PR DESCRIPTION
**Pull request by creativeappdevamit**
# What and why?

# How to test?

# Trello
https://trello.com/c/ITyML5G9/732-replace-rabbit-with-rest-call-ras-collection-instrument-to-rm-collection-exercise-service

### CODE REVIEW

File: src/main/java/uk/gov/ons/ctp/response/collection/exercise/endpoint/CollectionExerciseInstrumentReceiverEndpoint.java [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+  @RequestMapping(value = "/link", method = RequestMethod.POST)
+  public ResponseEntity<?> collectionInstrumentLink(
+      final @RequestBody @Valid CollectionInstrumentMessageDTO collectionInstrumentMessageDTO)
+      throws CTPException {
+    log.with(collectionInstrumentMessageDTO.getInstrumentId())
+        .with(collectionInstrumentMessageDTO.getExerciseId())
+        .info("Consumed message");
+
+    UUID collectionExerciseId = collectionInstrumentMessageDTO.getExerciseId();
+
```
Comment (warrenbailey) [inline src/main/java/uk/gov/ons/ctp/response/collection/exercise/endpoint/CollectionExerciseInstrumentReceiverEndpoint.java:39]: just so i'm clear, collection instrument sends the instrument id to CE but it does nothing with it?

**Review by warrenbailey — CHANGES_REQUESTED** [review]
A couple of questions

File: src/main/java/uk/gov/ons/ctp/response/collection/exercise/message/CollectionInstrumentLoadedReceiver.java [hunk 2]
```diff
@@ -21,8 +20,8 @@
   public void acceptSampleUnit(CollectionInstrumentMessageDTO message) throws CTPException {
     log.with(message).debug("Consumed message");
 
-    UUID collexId = message.getExerciseId();
+    // UUID collexId = message.getExerciseId();
```
Comment (warrenbailey) [inline src/main/java/uk/gov/ons/ctp/response/collection/exercise/message/CollectionInstrumentLoadedReceiver.java:24]: Do we need this class anymore if all it does is log?

File: src/main/java/uk/gov/ons/ctp/response/collection/exercise/endpoint/CollectionExerciseInstrumentReceiverEndpoint.java (hunk 1 above)
Comment (creativeappdevamit) [reply src/main/java/uk/gov/ons/ctp/response/collection/exercise/endpoint/CollectionExerciseInstrumentReceiverEndpoint.java:39]: Seems like it only uses it for logging against which collection instrument id its doing the transition change.
The actual instrument Id is registered as a part of the sample unit communication from sample service.

File: src/main/java/uk/gov/ons/ctp/response/collection/exercise/message/CollectionInstrumentLoadedReceiver.java (hunk 2 above)
Comment (creativeappdevamit) [reply src/main/java/uk/gov/ons/ctp/response/collection/exercise/message/CollectionInstrumentLoadedReceiver.java:24]: We don't will take it out.

**Review by warrenbailey — APPROVED** [review]

**Pull request merged**